crates/holochain_types/src/dna/dna_manifest/dna_manifest_v1.rs

```diff
@@ -80,11 +80,31 @@ pub struct DnaManifestV1 {
     /// Only this affects the [`DnaHash`].
     pub integrity: IntegrityManifest,
 
-    #[serde(default)]
     /// Coordinator zomes to install with this DNA.
     ///
     /// Does not affect the [`DnaHash`].
+    #[serde(default)]
     pub coordinator: CoordinatorManifest,
+
+    /// A list of past "ancestors" of this DNA.
+    ///
+    /// Whenever a DNA is created which is intended to be used as a migration from
+    /// a previous DNA, the lineage should be updated to include the hash of the
+    /// DNA being migrated from. DNA hashes may also be removed from this list if
+    /// it is desired to remove them from the lineage.
```

Comment:
It's good context to have, but DNAs (and their manifests) have no notion of a publisher or even of an app, so that concept lives at two levels higher in the stack of composition and I'm not sure it belongs here.